content/en/apps/guides/hosting/4.x/self-hosting/_index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Self Hosting in CHT 4.x"
+linkTitle: "Self Hosting"
+weight: 10
+description: >
+  Details for hosting the CHT on self run infrastructure
+---
+
+# Recommendations and considerations
+
+## Multi vs Single node couchdb requirements
+
+For smaller deployments a [single node CouchDB][single-couch] instance can be used, for larger deployments a [multi-node CouchDB][multi-couch] cluster is generally recommended
+
+| Consideration                                                                   | [Single node CouchDB][single-couch] | [Multi-node clustered CouchDB][multi-couch] |
+| ------------------------------------------------------------------------------- | ----------------------------------- | ------------------------------------------- |
+| Less than 2000 users                                                            | {{< icon/yes >}}                    | {{< icon/yes >}}                            |
+| More than 2000 users                                                            | {{< icon/no >}}                     | {{< icon/yes >}}                            |
+| Less than 1 million documents                                                   | {{< icon/yes >}}                    | {{< icon/yes >}}                            |
+| More than 1 million documents                                                   | {{< icon/no >}}                     | {{< icon/yes >}}                            |
+| Significant bursts of traffic (i.e. "Sync day" for a lot of users once a month) | {{< icon/no >}}                     | {{< icon/yes >}}                            |
```

Comment:
"sync day" applies to most instances so this would rule out single node for just about everyone. I think we should remove this line and focus on the docs/day measure instead.